department-for-transport-BODS/bods
# feat: update IAVL stats endpoint

To test:
- Ensure `AVL_CONSUMER_API_BASE_URL` env var is pointing to AVL test public api
- Spin up local env
- Go to django admin panel --> periodic tasks
- Run the task: `save_operational_stats`
- Check celeryworker logs to see if it ran successfully
- Check db table `site_admin_operationalstats` - vehicle_count column should be populated
